### PR TITLE
Add first implementation of a visibility plot

### PR DESCRIPTION
**Pull request by j-faria**
This is heavily based on PyAstronomy.pyasl.transitVisibilityPlot and seems to match the output of STARALT (issue #2). But I need to check the airmass values.  
I would like to add more stuff like rise, set and twilight times.

### CODE REVIEW

File: visibility.py [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+  else:
+    plt.show()
+
+
+
+if __name__ == '__main__':
+  import datetime as dt
+
+  ids = ['HD41248', 'HD219828']
+  targets = []
```
Comment (DanielAndreasen) [inline visibility.py]: Can be written as `targets = [{'name': i, 'coord': SkyCoord.from_name(i)} for i in ids]` :) But not important.

**Comment by DanielAndreasen** [comment]
Cool, that works very well. If we make this to a script, so the user can input some targets like this

    python visibility.py HD1 HD2 HD3

This can be done with `argparse` easily (I can help with that). But let's also add some options, like observatory and date (can't think of anything else).

**Comment by j-faria** [comment]
Yap, adding a cli would be cool :+1:  sorry about the typo in the name of the branch...

**Comment by DanielAndreasen** [comment]
That's okay. It's already a very nice cool (just used it a little). So I guess I should merge when you add the cli part (I also want to test how this merging works in depth).

File: visibility.py [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+  for i in ids:
+    coord = SkyCoord.from_name(i)
```
Comment (DanielAndreasen) [inline visibility.py]: I think we should add a `try/except` here. It's easy to write the name wrong...

    try:
        coord = SkyCoord.from_name(i)
    except:
        print('Could not find target: %s' % i)

**Comment by j-faria** [comment]
Example output from 
`python visibility.py HD41248,HD219828 -s esolasilla -d 2016-04-04`


example

**Comment by DanielAndreasen** [comment]
What do you think about my changes? Try run it with the `-h` flag. There are a lot of possible observatories, so it is a bit crowded.

**Comment by j-faria** [comment]
It gets a bit confusing with all the options... How about

>     parser.add_argument('-s', '--site', default='esolasilla', 
>                         help='Observatory. Default is ESO La Silla. '
>                              'Common codes are esoparanal, lapalma, keck, lco, Palomar, etc')

and then still do the check afterwards?

**Comment by DanielAndreasen** [comment]
Sure, I agree it gets too confusing. The user has to give an observatory which is not in the list. I guess it's okay. If you fix it, I merge it :)

**Pull request merged**